robottelo/utils/url.py

```diff
@@ -1,9 +1,55 @@
 from urllib.parse import urlparse
 
+from robottelo.logging import logger
+
 
 def is_url(url):
     try:
         result = urlparse(url)
         return all([result.scheme, result.netloc])
     except (ValueError, AttributeError):
         return False
+
+
+def is_ipv4_url(text):
+    # Did not find the better way to filter only URLs so skipping it simple
+    # and open for reviewers suggestions
+    if isinstance(text, str) and '-ipv4' in text:
+        return True
+    return False
+
+
+def ipv6_translator(settings_list, setting_major, settings, data):
+    """Translates the hostname containing ipv4 to ipv6 and updates the settings object"""
+    dotted_settings = '.'.join(setting_major)
```

Comment:
Solution 2 is already in place and to do that we changing it to dot connected settings. I would rather change the setting to be named string(for unifirmality) instead of number or change the interger to string. 

@Griffin-Sullivan thoughts ?

Reply:
Sorry I should check my direct mentions more often.

So to clarify this is a problem with the fam.yaml settings because it uses this notation?

```yaml
FAM:
...
  COMPUTE_PROFILE:
      libvirt:
        ...
          compute_profile:
            ...
            attrs:
              ...
              nics_attributes:
                0:
                  ...
```

The reason I have it like this is because that's how https://github.com/theforeman/foreman-ansible-modules/blob/develop/tests/test_playbooks/vars/compute_profile.yml is set up. If we want to change the notation we just need to update FAM as well. Let me know if I need to make the changes.

Reply:
@Griffin-Sullivan Yes, Good to have uniformed behavior across robottelo.